Templates/Ros2FleetRobotTemplate/docs/scene_creation.md

```diff
@@ -0,0 +1,57 @@
+
+## Modifying the warehouse scene
+
+If you have the `WarehouseScene` prefab on your scene you can easily apply modifications. It is possible to alter virtually any object on the scene, remove it or add a new one. The structure of the `WarehouseScene` prefab contains a few levels of nested prefabs (please refer to [this document](../../../Gems/WarehouseAssets/README.md) for a description). For example, prefabs with boxes are located inside prefabs with storage racks, which are often located in storage packs (containing 4 racks). See the image below.
+
+![](images/WarehouseSceneSampleTree.png)
+
+You can alter objects on each level by removing, scaling or moving any objects. For example, you can:
+
+- Re-arrange boxes on shelves by translating and rotating existing ones, or adding new ones,
+- Remove boxes from a shelf to make places for different objects,
+- Remove the whole storage rack,
+- Translate racks to different locations.
+
+Another type of modification concerns the warehouse structure. As described in the [Warehouse Assets Gem README Tips](../../../Gems/WarehouseAssets/README.md), you can easily change frontal walls by replacing the solid wall with open gates. This way you can stack two or more warehouses, one after the other.
+
+![](images/stacked_warehouses1.png)
+
+![](images/stacked_warehouses2.png)
+
+> Note: in O3DE there are 2 ways you can alter instantiated prefab: by changing prefab instance or prefab definition (source). If you modify prefab source, all instances will be altered. Thus unless you really know what you are doing, you should edit instances only. Simple hint: do not double-click on the prefab in the Entity Outliner. When double-clicking the editor enters focused mode, prefab is highlighted in blue, changes will be saved in the prefab source. Please refer to [this document](https://www.o3de.org/docs/learning-guide/tutorials/entities-and-prefabs/entity-and-prefab-basics/) to learn about working with prefabs.
+
+## Creating the warehouse scene
+
+According to your needs, you may also create a custom warehouse using selected assets from the Warehouse Assets Gem. As described in the [Warehouse Assets Gem README](../../../Gems/WarehouseAssets/README.md), the gem contains structural assets defining the building and various objects to be placed inside. Structural assets include 3 prefabs:
+
+- Floor,
+- Walls with gates,
+- Roof with lights.
+
+If you intend to use them together, be sure to place them with the same transform, to ensure alignment. Besides the building structure, the gem provides a set of different objects to be used in the warehouse. These are arranged into prefabs with different levels of nesting. You can import the empty rack and place boxes on it, but you can also instantiate ready-to-use prefab with racks and boxes or pack of 4 racks. Of course, nothing prevents you from modifying such prefabs after instantiating.
```

Comment:
> If you intend to use them together, be sure to place them with the same transform, to ensure alignment.

What do you mean here? Setting the same values to the transform component or putting them in the same parent?